CHANGELOG.md

```diff
@@ -1,357 +1,248 @@
 # Change Log
 
-## Later releases
+## [v0.6.0](https://github.com/confluentinc/ksql/releases/tag/v0.6.0-ksqldb) (2019-11-19)
+
+### Features
```

Comment:
I'd of thought this list would only reflect changes made post 5.4.x, but it seems to hold more than this, e.g. changes from July.  Is this intentional?

Reply:
This changelog contains all changes since 5.3, as 5.4 hasn't been released yet. Do we prefer having only changes since 5.4 instead?

Reply:
I'm not sure.  No biggie though. Just thought I'd call it out in case it wasn't intentional